volatility3/framework/plugins/windows/ldrmodules.py

```diff
@@ -32,6 +32,12 @@ def get_requirements(cls):
                 description="Process IDs to include (all other processes are excluded)",
                 optional=True,
             ),
+            requirements.BooleanRequirement(
+                name="verbose",
```

Comment:
This feels like it might get confusing given the main CLI parameters of `-v` (which can also be written as `--verbose`).  I'd suggest something like `--detail` or `--extra` or `--extra-information` or even just `--full-path`.

Reply:
All right, I accepted your comments.